### PR TITLE
Admin cannot delete herself

### PR DESCRIPTION
**Pull request by shohre**
Delete button is not shown to the logged in admin on the admin list table
Read me files are updated for backend and frontend

### CODE REVIEW

File: frontend/src/admin/reducers/__tests__/userReducers-test.js [hunk 1]
```diff
@@ -12,11 +12,32 @@ describe('user reducers', () => {
...
+          somethingElse: 'another thing',
         },
       });
       expect(reduction).toEqual('SUPER');
     });
   });
+
+  describe('email', () => {
+    it('defaults to an empty string', () => {
+      const reduction = type(undefined, { email: 'donald@trump.com' });
```
Comment (camjackson) [inline frontend/src/admin/reducers/__tests__/userReducers-test.js:25]: Interesting choice of email address... 🤔

**Comment by camjackson** [comment]
Literally this: https://twitter.com/_dte/status/1000803384469803008

**Comment by camjackson** [comment]
Seriously though nice work 🙂 👍

File: frontend/src/admin/reducers/__tests__/userReducers-test.js (hunk 1 above)
Comment (shohre) [reply frontend/src/admin/reducers/__tests__/userReducers-test.js:25]: : )) He was the person on my mind at the time, for obvious reasons... :'(

**Pull request merged**